data/uhc_pack/functions/pre_generation/stage/step/teleport/dynamic/do.mcfunction

```diff
@@ -0,0 +1,4 @@
+forceload add 0 0
```

Comment:
So you are 'leaving behind' a force-loaded chunk in each dimension?

Reply:
...Sort of. I don't spawn the armor stands at the lobby center at the moment, just at 0 0. The chunk gets cleaned up later.
I'll move the place the armor stands spawn at to the lobby center, that will clean up some other comments you left as well